index.html

```diff
@@ -114,6 +114,96 @@ <h3 class="skills">Skills</h3>
             </li>
          </ul>
       </div>
-   </section> 
+   </section>
+
+   <section class="recent-work">
+      <h1 class="rw">MY RECENT WORK</h1>
+      <div class="divider"></div>
+      <div class="work1">
+         <div class="img1">
+            <img src="images/img.png" class="image" alt="software-portfolio">

```

Comment:
- [ ] Please make sure your images are being displayed on the screen. Right now this one is not being shown, it's showing the `alt` text instead. Same with all work images.